### DISCUSSION TITLE
Vite Inertia Documentation Issue

### DISCUSSION THREAD

**Discussion by yogeshgalav** [laravel/framework#46885] [General]
The Laravel documentation suggests using Inertia in following manner
https://laravel.com/docs/10.x/vite#inertia
```
import { resolvePageComponent } from 'laravel-vite-plugin/inertia-helpers'

createInertiaApp({
	resolve: (name) => {
		const page = resolvePageComponent('../views/pages/'+name, import.meta.glob('../views/pages/**/*.vue'));
		// page.then((module) => {
                //     module.layout =  module.layout || AppLayout;
               // });
        return page;
	},
    });
```

The above code suggests using resolvePageComponent from 'laravel-vite-plugin/inertia-helpers',
while it was better to use resolvePageComponent from 'vite-plugin-laravel/inertia' 
as it allows layout binding and component name without mentioning path
```
import { resolvePageComponent } from 'vite-plugin-laravel/inertia'

createInertiaApp({
	resolve: (name) => {
		const page = resolvePageComponent(name, import.meta.glob('../views/pages/**/*.vue'));
		page.then((module) => {
            module.layout =  module.layout || AppLayout;
        });

        return page;
	},
```

I don't know the package difference but can someone look into package difference and mention the better one in documentation.

**Comment by perisicnikola37** [comment]


**Reply by bram-pkg** [reply]
Was this written with GPT? It certainly seems like it.

**Reply by yogeshgalav** [reply]
Problem was real, I took help from GPT and then copied the content from there.
